docs/src/main/sphinx/connector/elasticsearch.rst

```diff
@@ -437,3 +437,20 @@ Property Name                                    Description
 ``elasticsearch.auth.user``                      User name to use to connect to Elasticsearch.
 ``elasticsearch.auth.password``                  Password to use to connect to Elasticsearch.
 ================================================ ==================================================================
+
+.. _elasticsearch-sql-support:
+
+SQL support
+-----------
+
+The connector provides read and write access to data and metadata in an
+Elasticsearch catalog. In addition to the :ref:`globally available
+<sql-globally-available>` and :ref:`read operation <sql-read-operations>`
+statements, the connector supports the following features:
+
+* :doc:`/sql/insert`
+* :doc:`/sql/create-table`
+* :doc:`/sql/create-table-as`
+* :doc:`/sql/drop-table`
+* :doc:`/sql/create-schema`
+* :doc:`/sql/drop-schema`
```

Comment:
Elasticsearch connector doesn't support these statements.

Reply:
Yes. Read-onyl.

Reply:
Thanks folks, I updated the text to read-only.

Reply:
Actually, there is an old WIP PR #3480 that supports CTAS & INSERT in Elasticsearch connector. It would be great if someone could finish the work.